asciidoc/cypher-workflow.adoc

```diff
@@ -489,52 +522,48 @@ Inbound conversion is carried out using http://cldr.unicode.org/development/deve
 | Path | *`Path`*
 |===
 
-+*+ JavaScript has no native integer type so a custom type is provided. For convenience, this can be disabled through configuration so that the native Number type is used instead. Note that this can lead to a loss of precision.
++*+ JavaScript has *_no native Integer type_* so a custom type is provided. For convenience, this can be disabled through configuration so that the *_native Number type_* is used instead. Note that this can lead to a loss of precision.
 ======
 
-// [.include-with-python]
-// ======
-// [options="header", cols="m,,"]
-// |===
-// | Neo4j type        | [deprecated]#Python 2 type# | Python 3 type
-// | null | `None` | `None`
-// | List | `list` | `list`
-// | Map | `dict` | `dict`
-// | Boolean | `bool` | `bool`
-// | Integer | `int / long`* | `int`
-// | Float | `float` | `float`
-// | String | `unicode`** | `str`
-// | ByteArray | `bytearray` | `bytearray`
-// | Date | *`neotime.Date`* | *`neotime.Date`*
-// | Time | *`neotime.Time`*† | *`neotime.Time`*†
-// | LocalTime | *`neotime.Time`*†† | *`neotime.Time`*††
-// | DateTime | *`neotime.DateTime`*† | *`neotime.DateTime`*†
-// | LocalDateTime | *`neotime.DateTime`*†† | *`neotime.DateTime`*††
-// | Duration | *`neotime.Duration`**** | *`neotime.Duration`****
-// | Point | *`Point`* | *`Point`*
-// | Node | *`Node`* | *`Node`*
-// | Relationship | *`Relationship`* | *`Relationship`*
-// | Path | *`Path`* | *`Path`*
-// |===
-//
-// +*+ While Cypher uses 64-bit signed integers, `int` can only hold integers up to `sys.maxint` in Python 2; `long` is used for values above this.
-//
-// +**+ In Python 2, a `str` passed as a parameter will always be implicitly converted to `unicode` via UTF-8.
-//
-// +***+ A `timedelta` passed as a parameter will always be implicitly converted to `neotime.Duration`.
-//
-// † Where tzinfo is not None.
-//
-// †† Where tzinfo is None.
-// ======
+[.include-with-python]
+======
+[options="header", cols="m,"]
+|===
+| Neo4j Type | Python 3 Type
```

Comment:
again, `Type` here is capitalised. I see that it is consistent with other languages, so perhaps we can ignore and docs team can fix at a later date if it is incorrect.